test/other/embind_tsgen_ignore_2.d.ts

```diff
@@ -88,7 +90,7 @@ export interface DerivedClass extends BaseClass {
 export type ValArr = [ number, number, number ];
 
 interface EmbindModule {
-  Test: {staticFunction(_0: number): number; staticFunctionWithParam(x: number): number; staticProperty: number};
+  Test: {staticFunction(_0: number): number; staticFunctionWithParam(x: number): number; staticProperty: number; get staticStringProperty(): string; set staticStringProperty(value: EmbindString)};
```

Comment:
Oops, these look like they still lack newlines.

Reply:
I fixed class properties new lines, but not the static properties. I forget which one it was, but this is the way that  TSC auto formats it or maybe it was to match typescripten. It definitely is getting unruly though, so I think I'll break from them and put it on a new line.

Reply:
Oh.. I hadn't seen that this was all inside the same curly braces.   Seems odd that TSC wouldn't use line breaks..

Reply:
I updated it to add new lines.